### PR TITLE
a very simple loop based outgoingEnqueueCombi base

### PR DESCRIPTION
**Pull request by behrad**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -175,6 +175,26 @@ CachedPersistence.prototype.cleanSubscriptions = function (client, cb) {
...
+  function finish (err) {
+    count++
+    if (err) {
+      errord = err
+      return cb(err)
+    }
+    if (count === subs.length && !errord) {
+      cb()
+    }
+  }
```
Comment (mcollina) [inline index.js]: This is going to be pretty hot. Can you use fastparallel?

Comment (behrad) [inline index.js]: only for persistences that don't override outgoingEnqueueCombi ! BUT for sure :)

Comment (mcollina) [inline index.js]: I would like this to stay on par with what we had in aedes.

Comment (behrad) [inline index.js]: Does it look good now @mcollina ?

**Comment by mcollina** [comment]
I've just noticed that there are no test for `outgoingEnqueueCombi`, can you add them in another PR?

**Comment by behrad** [comment]
Sure, I should update aedes-persistence tests

**Comment by behrad** [comment]
https://github.com/mcollina/aedes-persistence/pull/15 @mcollina

**Comment by mcollina** [comment]
The tests added in aedes-persistence 5.0.1 fails here, I can't release.

**Comment by behrad** [comment]
let me check it

**Comment by behrad** [comment]
I was trying to use aedes' fastparallel instance! However I think I should create one here

**Comment by behrad** [comment]
https://github.com/mcollina/aedes-cached-persistence/pull/10 @mcollina

**Pull request merged**